modules/claims_api/spec/requests/v1/intent_to_file_request_spec.rb

```diff
@@ -162,6 +162,21 @@
       end
     end
 
+    it 'doesn\'t allow additional fields' do
+      with_okta_user(scopes) do |auth_header|
+        params = data
+        params[:data][:attributes]['someBadField'] = 'someValue'
+
+        post "#{path}/validate", params: params.to_json, headers: headers.merge(auth_header)
```

Comment:
Just curious, but why test this in the `/validate` context?
That endpoint is "technically" deprecated right now and if we were to eventually remove the endopint altogether, we'd lose this test.

Reply:
ahh that was just an oversight. I think this was one of the last tests I added and just wasn't paying attention since it was mostly copy-paste-modify at that point. I'll move into into the standard post where it makes more sense